test/e2e/networking.go

```
@@ -123,12 +124,9 @@ var _ = Describe("Networking", func() {
 		defer func() {
 			defer GinkgoRecover()
 			By("Cleaning up the replication controller")
-			rc.Spec.Replicas = 0
-			rc, err = c.ReplicationControllers(ns).Update(rc)
-			if err != nil {
-				Fail(fmt.Sprintf("unable to modify replica count for rc %v: %v", rc.Name, err))
-			}
-			if err = c.ReplicationControllers(ns).Delete(rc.Name); err != nil {
+			// Resize the replication controller to zero to get rid of pods.
+			rcReaper, _ := kubectl.ReaperFor("ReplicationController", c)
```

Don't ignore the error value. Same below.